test/xskbench/um/xskbench_user.h

```diff
@@ -0,0 +1,193 @@
+//
+// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like XDP needs "CxPlat" abstractions now.

Reply:
Yep. As discussed in the other thread, I'll copy any CxPlat abstractions needed from msquic and we can take a dependency on the cxplat repo when it is ready